tests/unit/document/test_database_subindices.py

```diff
@@ -0,0 +1,33 @@
+from docarray import Document, DocumentArray, dataclass
+from docarray.typing import Image, Text
+
+@dataclass
+class MyDocument:
+    image: Image
+    paragraph: Text
+
+def test_database_subindices():
+    # extend with Documents, including embeddings
+    _docs = [(MyDocument( image='https://docarray.jina.ai/_images/apple.png', paragraph='hello world'))]
```

Comment:
make sure to add a local dependency, this test may fail in case some documentation change happens in the repo, or docarray website is down ...